integrations/mixpanel/lib/index.js

```diff
@@ -316,6 +317,32 @@ Mixpanel.prototype.alias = function(alias) {
   mp.alias(to, alias.from());
 };
 
+/**
+ *
+ * @param {Group} group
+ */
+
+Mixpanel.prototype.group = function(group) {
+  var groupIdentifierTraits = this.options.groupIdentifierTraits;
+  var groupId = group.groupId();
+  var userId = this.analytics.user().id();
+  var traits = group.properties();
+  if (!groupId || !userId || !groupIdentifierTraits.length) {
```

Comment:
Prob should not call `length` until we know whether `groupIdentifierTraits` is not null.

Reply:
Uhm, good point. I think it's defined by default on the `option` function.